src/Components/Search/SearchBar.tsx

```diff
@@ -179,6 +191,7 @@ export class SearchBar extends Component<Props, State> {
     return (
       <AutosuggestContainer>
         <Autosuggest
+          alwaysRenderSuggestions={typeof jest === "undefined" ? false : true}
```

Comment:
This is the hack...I couldn't figure out how to render suggestions from tests (even though everything is being passed in properly). cc @damassi maybe we can pair on figuring out this lib testing wise?

Reply:
Might be worth moving this directly into `AutoSuggest` -- meaning, `alwaysRenderSuggestions` prop takes normal true / false / app flow stuff -- and then from inside the component you override the result if in a test environment. That said, this _should_ just work since we're rendering inside of a browser environment (jsdom). Might be something up elsewhere.

Reply:
if we're keeping a hack, it probably makes sense in our code/clients vs the actual lib. But I'd love to understand why this hack is needed.